src/test_l1_hdmi_cec_driver.c

```diff
@@ -954,7 +963,7 @@ void test_hdmicec_hal_l1_removeLogicalAddress_sourceDevice( void )
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
     result = HdmiCecRemoveLogicalAddress(handle, logicalAddress );
-    if (HDMI_CEC_IO_NOT_OPENED != result) { UT_FAIL("HdmiCecRemoveLogicalAddress failed"); }
+    CHECK_FOR_EXTENDED_ERROR_CODE(result,HDMI_CEC_IO_NOT_OPENED,HDMI_CEC_IO_INVALID_ARGUMENT)
```

Comment:
As per new interface file, these functions should return NOT SUPPORTED from the source device. check for not supported.